### PR TITLE
Fix Conv2D for case of group for DepthwiseConv

### PR DESCRIPTION
**Pull request by seanshpark**
This will try to convert to DepthwiseConv with condition of group value.
- based on https://github.com/onnx/onnx-tensorflow/issues/470#issuecomment-681584457 by CyanHillFox@github

Signed-off-by: SaeHie Park <saehie.park@samsung.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
I've go this error in one of the CI but I don't know how to reproduce this... can anyone help how to reproduce or how to fix?
```
ERROR: test_identity_sequence_cpu (backend.test_onnx_backend.OnnxBackendNodeModelTest)
```

**Comment by chinhuang007** [comment]
@seanshpark Thanks for the contribution! The CI problem is introduced by a recently merged PR in ONNX. I already created a patch, https://github.com/onnx/onnx-tensorflow/pull/868. So please don't worry about it.

File: onnx_tf/handlers/backend/conv_mixin.py [hunk 1]
```diff
@@ -91,16 +91,47 @@ def conv(cls, node, input_dict, transpose=False):
...
-    weight_groups = tf.split(weights, num_or_size_splits=group, axis=-1)
+    if depthwise is True:
+      # Depthwise convolution.
+      # The convolution kernel layout in tf.depthwise_conv is:
+      # [filter_height, filter_width, in_channels, channel_multiplier]
+      # Weight is now (KH x KW X C/g X M), or more precisely, (KH x KW X C/g X (g * M/g)),
+      # we reshape it to (KH x KW x C x M/g)
+      # NOTE: Assuming weight has fixed shape.
+      if weight_shape is None or None in weight_shape or group is None:
+        depthwise = False
```
Comment (chinhuang007) [inline onnx_tf/handlers/backend/conv_mixin.py:107]: Is it possible to combine code to determine whether depthwise in True? For example, if len(weight_shape) == 4, then weight_shape is not None. And looks like the logic for group could be simplified too.

Comment (seanshpark) [reply onnx_tf/handlers/backend/conv_mixin.py:107]: Thanks for the review! I'll revise the change :)

**Comment by seanshpark** [comment]
@chinhuang007 , I'm not that good at python so better suggestion are welcome :)

**Comment by seanshpark** [comment]
If the CI fails, I'll rebase after #868 lands

File: onnx_tf/handlers/backend/conv_mixin.py [hunk 2]
```diff
@@ -224,16 +253,35 @@ def conv(cls, node, input_dict, transpose=False):
...
-          for (x, weight) in zip(xs, weight_groups)
-      ]
+    else: # not transpose:
+      if depthwise is True:
+        if compute_format == "NHWC":
+          strides = [1] + strides + [1]
+        elif compute_format == 'NCHW':
+          strides = [1, 1] + strides
+        else:
+          raise ValueError("Invalid storage_format: {}".format(storage_format))
```
Comment (chinhuang007) [inline onnx_tf/handlers/backend/conv_mixin.py:263]: Should it be invalid compute_format?

Comment (seanshpark) [reply onnx_tf/handlers/backend/conv_mixin.py:263]: Thanks!

**Review by chinhuang007 — APPROVED** [review]
LGMT!

**Pull request merged**